uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1005.php

```diff
@@ -21,7 +21,7 @@ public function upgrade_1005() {
     ];
     
     foreach ($menuItems as $menuItem) {
-      $menuItem->label = 'Absences';
+      $menuItem->label = 'Leave';
```

Comment:
But this won't affect new sites, you need a new upgrader for that

Reply:
The upgrader was already new 😄

Reply:
but it was already merged and deployed to staging in another PR, so staging won't be affected by this or am I missing something ?

Reply:
@omarabuhussein it hasn't been merged to staging, but to a parent branch: `PCHR-2579-admin-menu-updates`